src/Forms/Shared/Samples/Layers/ExportTiles/ExportTiles.xaml

```diff
@@ -13,10 +13,28 @@
         <Grid Grid.Row="0" BackgroundColor="MediumVioletRed">
             <esriUI:MapView x:Name="MyMapView" IsVisible="True" />
         </Grid>
-        <ActivityIndicator x:Name="MyProgressBar"
-                           Grid.Row="1"
-                           IsRunning="True"
-                           IsVisible="False" />
+        <Grid Grid.Row="1" Margin="5,0">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="320" />
+                <ColumnDefinition Width="auto" />
+            </Grid.ColumnDefinitions>
+            <ProgressBar x:Name="MyProgressBar"
```

Comment:
Could use a `Margin = "5"` so that the progress bar isnt touching the cancel button.